### PR TITLE
Update instructions in the Convert JS to TS (in VS) article

### PR DESCRIPTION
**Pull request by kbrandl**
Fixes issues: #317, #318, #319, and #320.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-VS-to-TS-317-thru-320/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810302109474791-527%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=owV5CG6EVLbFwoLMFIXwGO72yUXblxsY4JeIay1v70U%3D&st=2018-10-30T21%3A06%3A05Z&se=2018-11-30T21%3A11%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/convert-javascript-to-typescript.md [hunk 1]
```diff
@@ -79,12 +77,28 @@ This article shows you how to create an Excel add-in using Visual Studio and the
...
-12. In the **Home.ts** file, change **'1.1'** to **1.1** (that is, remove the quotation marks) in the following line, and save the file:
+12. In the **Home.ts** file, change **'1.1'** to **1.1** (that is, remove the quotation marks) in the following line:
 
 	```javascript
 	if (!Office.context.requirements.isSetSupported('ExcelApi', '1.1')) {
 	```
 
+13. In the **Home.ts** file, find the `displaySelectedCells` function, replace the entire function with the following code, and save the file:
+
+    ```javascript
```
Comment (AlexJerabek) [inline docs/develop/convert-javascript-to-typescript.md]: Shouldn't this be TypeScript?

File: docs/develop/convert-javascript-to-typescript.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-ms.date: 01/19/2018
+ms.date: 10/30/2018
 ---
 
 # Convert an Office Add-in project in Visual Studio to TypeScript
 
-You can use the Office Add-in template in Visual Studio to create an add-in that uses JavaScript, and then convert that add-in project to TypeScript. By using Visual Studio to create the add-in project, you avoid having to create your Office Add-in TypeScript project from scratch. 
-
-This article shows you how to create an Excel add-in using Visual Studio and then convert the add-in project from JavaScript to TypeScript. You can use the same process to convert other types of Office Add-in JavaScript projects to TypeScript in Visual Studio.
+You can use the Office Add-in template in Visual Studio to create an add-in that uses JavaScript, and then convert that add-in project to TypeScript. This article describes how to create an Excel add-in using Visual Studio and then convert the add-in project from JavaScript to TypeScript. You can use the same process to convert other types of Office Add-in projects from JavaScript to TypeScript in Visual Studio.
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/develop/convert-javascript-to-typescript.md]: Agreed; will update accordingly.

File: docs/develop/convert-javascript-to-typescript.md [hunk 3]
```diff
@@ -17,13 +15,13 @@ This article shows you how to create an Excel add-in using Visual Studio and the
...
 
-- TypeScript 2.3 for Visual Studio 2017
+- TypeScript SDK version 2.3 or later (for Visual Studio 2017)
 
-    > [!NOTE]
-    > TypeScript should be installed by default with Visual Studio 2017, but you can [use the Visual Studio Installer](https://docs.microsoft.com/visualstudio/install/modify-visual-studio) to confirm that it is installed. In the Visual Studio Installer, select the **Individual components** tab and then verify that **TypeScript 2.3 SDK** is selected under **SDKs, libraries, and frameworks**.
+    > [!TIP]
+    > In the [Visual Studio Installer](https://docs.microsoft.com/visualstudio/install/modify-visual-studio), select the **Individual components** tab and then scroll down to the **SDKs, libraries, and frameworks** section. Within that section, ensure that at least one of the **TypeScript SDK** components (version 2.3 or later) is selected. If none of the **TypeScript SDK** components are selected, select the latest available version of the SDK and then choose the **Modify** button to [install that individual component](https://docs.microsoft.com/en-us/visualstudio/install/modify-visual-studio?view=vs-2017#modify-individual-components).
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply docs/develop/convert-javascript-to-typescript.md]: Actually "ensure that..." is correct grammar. I've actually received this feedback from Laura Graham (a long-time MSFT editor on Kelly's team) multiple times in the past, but checked again with her now to confirm.  :)

File: docs/develop/convert-javascript-to-typescript.md (hunk 1 above)
Comment (kbrandl) [reply docs/develop/convert-javascript-to-typescript.md]: Yes!  Good catch; I'll update this and the other 2 snippets from Home.ts.

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the feedback! I've made some updates, per your feedback, so it's ready for you to take another look when you have a minute. Thanks again!

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-VS-to-TS-317-thru-320/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810312120523222-527%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eurYWOqizbAjooGcmUvHzAxBDn2cbORM3AisemFTPc4%3D&st=2018-10-31T21%3A17%3A05Z&se=2018-12-01T21%3A22%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**